core/src/main/java/org/elasticsearch/search/suggest/DirectSpellcheckerSettings.java

```diff
@@ -118,8 +118,8 @@ public int minWordLength() {
         return minWordLength;
     }
 
-    public void minQueryLength(int minQueryLength) {
-        this.minWordLength = minQueryLength;
+    public void minWordLength(int minWordLength) {
```

Comment:
I think this is a good renaming, since it is consistenc with the getter `minWordLength` and I think also with the naming in the TermSuggestionBuilder. 
Generally we can rename user facing API with major versions without deprecation I think, but better to keep track of it somewhere. At least thats what we did with the query builders. 
In this case after some inspection I don't even think it needs to go into the migrate doc, but also can't hurt to put a line there.